src/util/route_util.js

```diff
@@ -17,7 +17,7 @@ const Protected = ({ component: Component, path, loggedIn }) => (
      loggedIn ? (
       <Component {...props} />
     ) : (
-      <Redirect to="/signin" />
+      <Redirect to="/" />
```

Comment:
Both AuthRoute and ProtectedRoute redirect to the same page?

Reply:
Waiting to see what our logged-in landing page is - I've put in /home for now.